template/onchainKit/store/inMemoryStorageService.ts

```diff
@@ -0,0 +1,32 @@
+import { StorageInterface } from '../types';
+
+// InMemoryStorageService: A service to store data in JavaScript memory.
+// It's a simple key-value storage using a Map. The data is stored as long as the page is not reloaded.
+export class InMemoryStorageService implements StorageInterface {
```

Comment:
Not for this PR, BUT, I even wonder if we need this class, and we should just make this a functional-oriented code.

Mostly because Functional-orinted is more memory efficient. And also there is less risk to have this object with multiple instances.